server/implementation/src/main/java/io/smallrye/graphql/bootstrap/LogPayloadOption.java

```diff
@@ -0,0 +1,24 @@
+package io.smallrye.graphql.bootstrap;
+
+public enum LogPayloadOption {
+
+    OFF("off"),
+    TRUE("withoutvariable"),
+    FALSE("off"),
+    WITHOUTVARIABLE("withoutvariable"),
+    WITHVARIABLE("withvariable");
```

Comment:
I should have said this previously, but maybe if's easier to make the actual enum also lower case ? So 

```
    off("off"),
    true("withoutvariable"),
    false("off"),
    withoutvariable("withoutvariable"),
    withvariable("withvariable");
```